lib/hooks.dart

```diff
@@ -0,0 +1,101 @@
+@JS()
+library hooks;
+
+import 'package:js/js.dart';
+import 'package:react/react.dart';
+import 'package:react/react_client/react_interop.dart';
+
+/// The return value of [useState].
+///
+/// The current value of the state is available via [value] and
+/// functions to update it are available via [set] and [setWithUpdater].
+///
+/// Note are two rules for using Hooks (<https://reactjs.org/docs/hooks-rules.html>):
```

Comment:
#supertinytyponit I think there's a missing `there` in the comment, or something similar to that
```suggestion
/// Note there are two rules for using Hooks (<https://reactjs.org/docs/hooks-rules.html>):
```

Reply:
Good catch! Thanks!